### PR TITLE
add loggings for inline tab group

### PR DESCRIPTION
**Pull request by chassu**
Still WIP. I'm thinking about what kind of data the logging can generate. Open to any suggestions.

### CODE REVIEW

**Review by darinwf — APPROVED** [review]
LGTM. Would be great to get a review from Macy / Chung who know the logging system better.

**Review by chung-lai — APPROVED** [review]
LGTM, thanks for adding the logging!

File: Client/Application/SceneDelegate.swift [hunk 1]
```diff
@@ -100,9 +100,18 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         bvc.downloadQueue.resumeAll()
 
+        var attributes = EnvironmentHelper.shared.getAttributes()
```
Comment (chung-lai) [inline Client/Application/SceneDelegate.swift:103]: I think we had a conversation before about whether `normalTabsOpened` should include inline tab groups' tabs. Should we have `normalTabsOpened`, `numTabGroupsTotal` and `numofTabGroupTabs`? That way, we can see some distribution of # of tab groups' tabs.

Or we already log that when we open tab switcher?

**Comment by chassu** [comment]
Some new added loggings are depending on https://github.com/neevaco/neeva-ios/pull/2891, will wait for that to be merged then I'll merge this.

**Pull request merged**